benchopt/runner.py

```diff
@@ -355,7 +355,7 @@ def run_one_solver(benchmark, objective, solver, meta, max_runs, n_repetitions,
 def run_benchmark(benchmark, solver_names=None, forced_solvers=None,
                   dataset_names=None, objective_filters=None,
                   max_runs=10, n_repetitions=1, timeout=100,
-                  plot_result=True, show_progress=True, pdb=False):
+                  plot_result=True,html = True, show_progress=True, pdb=False):
```

Comment:
Add whitespace after a coma. not sure if there is a rational here but this is the chosen convention in pep8 :)
```suggestion
                  plot_result=True, html=True, show_progress=True, pdb=False):
```